code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -109,17 +109,11 @@
 /datum/reagent/medicine/cryoxadone/on_mob_life(mob/living/M)
 	if(M.stat != DEAD && M.bodytemperature < T0C) // Low temperatures are required to take effect.
 		M.status_flags &= ~DISFIGURED
-		M.adjustCloneLoss(-1)
-		M.adjustOxyLoss(-5)
-		M.adjustBruteLoss(-1)
-		M.adjustFireLoss(-1)
-		M.adjustToxLoss(-1)
-	if(M.stat != DEAD && M.bodytemperature < 225) // At lower temperatures (cryo) the full effect is boosted
-		M.adjustCloneLoss(-1)
-		M.adjustOxyLoss(-2)
-		M.adjustBruteLoss(-2)
-		M.adjustFireLoss(-2)
-		M.adjustToxLoss(-2)
+		M.adjustCloneLoss(-3)
```

Comment:
Previously, cryoxadone activated fully at t0c, but a freezer/cryotube change meant you reached t0c faster, so it buffed cryo. To even it out the temperature to get full healing was lowered to 225, but this nerfed ghettocryo and cryox+ice to hell and back. I raised the healing temp back to what it was but lowered the healing rate a bit, to hopefully keep cryo approximately the same.